src/rust/ensogl/lib/core/src/control/io/mouse.rs

```diff
@@ -130,6 +132,12 @@ macro_rules! define_bindings {
     };
 }
 
+fn event_listener_options() -> web_sys::AddEventListenerOptions {
+    let mut options = web_sys::AddEventListenerOptions::new();
+    options.capture(true).passive(false);
```

Comment:
could you add comments here what it realy does with a links to appropriate msdn docs? I watn to read about it and understand the reasoning behind this solution.